doc/rst/language/spec/tuples.rst

```diff
@@ -135,7 +135,7 @@ An underscore can be used to omit components when splitting a tuple (see
 
    defines three tuple variables. Variable ``x1`` is a 2-tuple with
    component types ``string`` and ``real``. It is initialized such that
-   the first component is ``"hello"`` and the second component is
+   the initial component is ``"hello"`` and the next component is
```

Comment:
This "initial", "next"/"other" language is starting to feel more contrived to me